tests/chunkers/test_base_chunker.py

```diff
@@ -0,0 +1,68 @@
+import hashlib
+import unittest
+from unittest.mock import MagicMock
+
+from embedchain.chunkers.base_chunker import BaseChunker
+from embedchain.models.data_type import DataType
+
+
+class TestBaseChunker(unittest.TestCase):
```

Comment:
Let's not use unittest here since it is quite limited. We should use `pytest` in general since it has way more extensions and is better than in-built unittest.